components/automate-ui/src/app/helpers/auth/validator.ts

```diff
@@ -0,0 +1,35 @@
+import { FormControl, FormGroup } from '@angular/forms';
```

Comment:
Since I was looking at the validators I took this opportunity for a quick cut and paste to DRY up some code. (So no new code here; this file was extracted from the deleted bits of code you'll find below.)